msos/MixedStack.cs

```diff
@@ -517,6 +517,13 @@ public UnifiedBlockingObject(ulong handle, string objectName, string objectType)
             Reason = ConvertToUnified(objectType);
         }
 
+        public UnifiedBlockingObject(long msTimeout) : this(BlockingObjectOrigin.StackWalker)
+        {
+            Type = UnifiedBlockingType.ThreadSleep;
+            Reason = UnifiedBlockingReason.ThreadWait;
+            ReasonDescription = $"MsTimeout : {msTimeout}";
```

Comment:
Please change to read "Sleep with timeout: 766ms" if there is a timeout, or to "Sleep with no timeout" if the timeout is infinite.